addons/controllers/clusterbootstrap_controller_test.go

```diff
@@ -17,6 +17,8 @@ import (
 	"sigs.k8s.io/cluster-api/util/secret"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/vmware-tanzu/tanzu-framework/addons/pkg/constants"
```

Comment:
suggestion: this import could be grouped with L22

Reply:
Fixed